debian/README.source

```diff
@@ -0,0 +1,51 @@
+Debian packaging of LinuxCNC/Mesaflash
+======================================
+
+The following steps were performed to transform Upstream's git repository
+into a source tarball and a debian subdirectory that shines for the
+main distribution. The major obstacle is that Mesaflash is used across
+a wide span of distributions, from Wheezy  (Jessie anyon?) up to the
+latest Ubuntu releases. Not too many changes are required for Debian
+unstable, the most prominent one likely is that there is already a debiah
+directory in the source tree.
+
+Steps
+-----
+
+1. Generate the source tarball
+
+   debian/rules get-orig-source
+
+
+2. Prepare shell to work in temp directory
+   export WORKDIR=$(mktemp -d)
+   mv mesaflash_*.orig.tar.xz ${WORKDIR}/
+   bash
+   cd $WORKDIR
+
+2. Unpack
+   tar xJvf mesaflash_*.orig.tar.xz
+
+3. Slightly adapt for unstable
+   bash
+   cd $WORKDIR/mesaflash
+   rm debian/compat
+   sed -i 's/debhelper .*,/debhelper-compat (= 13)/' debian/control 
```

Comment:
Should be `sed -i 's/debhelper .*,/debhelper-compat (= 13)/,' debian/control`, I think?  (Append a `,` to the end of the new line.)

Reply:
fixed